debug/hillshade_benchmark.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
```

Comment:
Benchmark files are typically put in the `bench` folder. Similar to #8645 this technique could be used to detect leaks in specific layers or styles - either by adding it to the existing benchmarking tooling or as a separate set of benchmarks. That work can follow in a separate PR.